apps/server/src/modules/learnroom/service/coursegroup.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@nestjs/common';
+import { CourseGroup, EntityId } from '@shared/domain';
+import { CourseGroupRepo } from '@shared/repo';
+
+@Injectable()
+export class CourseGroupService {
+	constructor(private readonly repo: CourseGroupRepo) {}
+
+	public async deleteUserDataFromCourseGroup(userId: EntityId): Promise<number> {
+		const [courseGroups, count] = await this.repo.findByUserId(userId);
+
+		const updatedCourseGroups = courseGroups.map(
+			(courseGroup: CourseGroup) =>
+				({
+					...courseGroup,
+					students: courseGroup.students.remove((u) => u.id === userId),
```

Comment:
Could you please add method for this operation in the `CourseGroup` class and move this logic there?

Reply:
done